src/content/tutorials/observability/data-exploration/accessing-grafana/_index.md

```diff
@@ -1,58 +1,79 @@
+  - How do I log in to Grafana?
+  - How do I explore metrics and logs?
+  - What is the Drilldown app in Grafana?
+  - How do I run queries in Explore?
 owner:
   - https://github.com/orgs/giantswarm/teams/team-atlas
 ---
 
-Giant Swarm provides an Observability Platform in each installation, based on [Grafana Mimir](https://grafana.com/oss/mimir/) for metrics, and [Grafana Loki](https://grafana.com/oss/loki/) for logs, collecting data from the system and apps managed by Giant Swarm, as well as any source of data you configure according to [the data ingestion tutorial]({{< relref "/tutorials/observability/data-ingestion" >}}).
+This guide shows you how to use the Explore and DrillDown features in Grafana to query and analyze the metrics and logs collected by the Giant Swarm Observability Platform. It also explains how to authenticate and access your Grafana instance securely.
 
-These metrics and logs are available to be explored in your [Grafana](https://grafana.com/) instance on the installation.
+Giant Swarm provides an Observability Platform in each installation, based on [Grafana Mimir](https://grafana.com/oss/mimir/) for metrics and [Grafana Loki](https://grafana.com/oss/loki/) for logs. The platform collects data from the system and apps managed by Giant Swarm, as well as any source of data you configure according to [the data ingestion tutorial]({{< relref "/tutorials/observability/data-ingestion" >}}).
 
-![Home dashboard](home-dashboard.png)
+You can use the [Explore](https://grafana.com/docs/grafana/latest/explore/) and DrillDown features in your [Grafana](https://grafana.com/) instance to query, visualize, and analyze your data interactively.
 
-We aim to provide a useful set of dashboards for you to quickly explore all observability data relevant to you, giving you access to the same information that we use for operations, support, and billing.
+![Grafana Explore tab](explore-tab.png)
 
-## Accessing dashboards
+## How to access and authenticate in Grafana
 
-The address for your installation's `grafana` instance is composed of the base domain plus the `grafana` subdomain.
+1. **Open Grafana:**
+   - Go to your installation's Grafana instance (for example, `https://grafana.my-company.gigantic.io/.com`).
+   - The address for your installation’s grafana instance is composed of the base domain plus the grafana subdomain. If you don't know the address, ask your Giant Swarm support contact.
 
-If you don't know the base domain for your installation, you can ask your Giant Swarm support contact for the address.
+2. **Log in with single sign-on (SSO):**
+   - Access to Grafana is controlled via SSO, using the same identity provider you use for the [platform API]({{< relref "/tutorials/access-management/authentication" >}}).
+   - To access Grafana as a customer, you must be a member of the admin group in your identity provider. This group specifies which users have admin permissions in the Giant Swarm installation.
```

Comment:
This is not the case anymore with organizations though

Reply:
So what's it now? "You must either be member of the admin group or any other group with access rights in your identity provider" or something like that?

Reply:
So today, I think all customers that can authenticate via dex are viewers :)

Reply:
How would you formulate it then? Or just remove it completely?

Reply:
I would maybe remove it completely yes because as long as you have access to dex then you can view our data by default.